### PR TITLE
Support qdrant cluster version update

### PR DESCRIPTION
**Pull request by mikheillomidzeq**
Support qdrant cluster version update. Move the version field to the proper location.

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -31,9 +31,9 @@ endif
 requirements:
 	go install github.com/goreleaser/goreleaser/v2@latest
 	go install github.com/mitchellh/gox@latest
-
-generate-client:
-	cd internal && swagger-codegen generate -i ./spec.json -l go --output client --additional-properties packageName=cloud
+# outdated
+#generate-client:
+#	cd internal && swagger-codegen generate -i ./spec.json -l go --output client --additional-properties packageName=cloud
```
Comment (Robert-Stam) [inline GNUmakefile:36]: what is different?

Comment (Robert-Stam) [inline GNUmakefile:36]: why not delete?

**Review by Robert-Stam — APPROVED** [review]
Small simplification, further LGTM

File: qdrant/resource_accounts_cluster.go [hunk 2]
```diff
@@ -82,9 +82,9 @@ func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, m interf
 		return diag.FromErr(fmt.Errorf("%s: %v", errorPrefix, err))
 	}
 	// Set required version to 'latest' if not provided
-	if cluster.Version == nil {
+	if cluster.Configuration.Version == nil || *cluster.Configuration.Version == "" {
```
[the remark this answers is not in the document]
Comment (mikheillomidzeq) [reply qdrant/resource_accounts_cluster.go:85]: Looks much nicer, thanks!

**Review by Robert-Stam — APPROVED** [review]
LGTM

**Pull request merged**